storage/streams/src/main/java/io/apicurio/registry/streams/StreamsRegistryConfiguration.java

```diff
@@ -97,6 +98,7 @@ public StreamsProperties streamsProperties(
                 empties = {"ssl.endpoint.identification.algorithm="}
         ) Properties properties
     ) {
+        log.debug("Providing a new ProducerActions<> instance.");
```

Comment:
Probably OK to leave in most of these debug() messages... but i'm also fine with removing them if you guys think that's best.

Reply:
+1 to leave the debug messages.